docs/content/doc/features/authentication.en-us.md

```diff
@@ -166,11 +166,47 @@ Uses the following fields:
 
 ## PAM (Pluggable Authentication Module)
 
-To configure PAM, set the 'PAM Service Name' to a filename in `/etc/pam.d/`. To
-work with normal Linux passwords, the user running Gitea must have read access
-to `/etc/shadow`.
-
-**Note**: PAM support is added via [build-time flags](https://docs.gitea.io/en-us/install-from-source/#build), and the official binaries provided do not have this enabled.
+This procedure enables PAM authentication.  Users may still be added to the
+system manually using the user administration.  PAM provides a mechanism to
+automatically add users to the current database by testing them against PAM
+authentication.  To work with normal Linux passwords, the user running Gitea
+must also have read access to `/etc/shadow` in order to check the validity of
+the account when logging in using a public key.
+
+**Note**: Once a user has been authenticated use of items such as SSH public
+keys _may_ bypass the login check system.  Therefore, if disabling a user, it
+is recommended to manually disable the account in Gitea using the built-in user
+manager as well.
```

Comment:
```suggestion
**Note**: If a user has added SSH public keys into Gitea, the use of these
keys _may_ bypass the login check system.  Therefore, if you wish to disable a user who
authenticates with PAM, you _should_ also manually disable the account in Gitea using the 
built-in user manager.
```

Reply:
I didn't want to state this because I _really_ don't know and would just be assuming (and you know where that gets people), but I'm guessing that this is also the reason that the application needs read access to the user file `/etc/shadow`.

Reply:
There is no check performed against PAM when a user logs in with ssh - so if it is intended for them to be disabled admins will need to do this disabling manually.